src/main/webapp/WEB-INF/web.xml

```diff
@@ -8,9 +8,20 @@
     </servlet>
 
     <!-- Маппинг сервлета на URL-паттерн -->
```

Comment:
What if some developer from England, Germany or India will check your code...
He will not understand this text...